src/coreclr/vm/amd64/AsmHelpers.asm

```diff
@@ -497,4 +497,14 @@ NESTED_ENTRY InterpreterStub, _TEXT
 NESTED_END InterpreterStub, _TEXT
 endif ; FEATURE_INTERPRETER
 
+; rcx -This pointer
+; rdx -ReturnBuffer
+LEAF_ENTRY ThisPtrRetBufPrecodeWorker, _TEXT
+    mov  METHODDESC_REGISTER, [METHODDESC_REGISTER + 0]
```

Comment:
A nit - use the `ThisPtrRetBufPrecodeData__Target` instead of 0 like in the Unix code?